src/util/output_channels.ts

```diff
@@ -70,21 +126,39 @@ export async function executeCommandInOutputChannels(command: CliCommand, name:
     // eslint-disable-next-line prefer-const
     startProcess = spawn(cmd.cliCommand, cmd.cliArguments);
     CACHED_CHILDPROCESS.set(name, startProcess);
-    startProcess.stdout.on('data', (chunk) => {
+
+    startProcess.stdout.on('data', async (chunk) => {
+      if (chunk.toString().includes('Please provide credentials for image registry')) {
```

Comment:
I think this will work just fine.

Just a note:
I am not absolutely sure that the sentence `Please provide credentials for image registry` won't be split into two chunks sometimes.

But it probably never happens, buffering is line based. Just something to keep in mind.

Reply:
now I am concatenating the chunk data and then checking the string.